tests/benchmarking/data.py

```diff
@@ -281,58 +281,57 @@
   "audiences": [
     {
       "conditions": "[\"and\", [\"or\", [\"or\", {\"name\": \"browser_type\", "
-                    "\"type\": \"custom_dimension\", \"value\": \"safari\"}]]]",
```

Comment:
These are correct as this benchmarking data is from our legacy product. Please restore all this.